vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -231,53 +231,53 @@ export default class SbcHeader extends NavigationMixin {
   logout () {
     if (this.redirectOnLogout) {
       const url = encodeURIComponent(this.redirectOnLogout)
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signout/${url}`)
+      window.location.assign(`${this.getContextPath()}signout/${url}`)
     } else {
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signout`)
+      window.location.assign(`${this.getContextPath()}signout`)
     }
   }
 
   login () {
     if (this.redirectOnLoginSuccess) {
       let url = encodeURIComponent(this.redirectOnLoginSuccess)
       url += this.redirectOnLoginFail ? `/${encodeURIComponent(this.redirectOnLoginFail)}` : ''
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signin/bcsc/${url}`)
+      window.location.assign(`${this.getContextPath()}signin/bcsc/${url}`)
     } else {
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signin/bcsc`)
+      window.location.assign(`${this.getContextPath()}signin/bcsc`)
     }
   }
 
   private goToHome () {
     if (this.inAuth) {
-      this.navigateTo(ConfigHelper.getAuthContextPath(), '/home')
```

Comment:
Except signin and signout all other routes should be pointing to auth

Reply:
done